notebooks/format_benchmark.py

```diff
@@ -0,0 +1,169 @@
+import os
+import re
+import simplejson as json
+from dateutil import parser
+
+
+def convert_benchmark(output_dir, monitoring_log):
+    """
+    Converts and Saves a downloaded monitoring log to a formated JSON
```

Comment:
Could you add a link to the monitoring script this code processes? Since it won't work for a general log.